QuickChart/QuickChart.cs

```diff
@@ -31,17 +31,26 @@ public class Chart
 
         private static readonly HttpClient Client = new HttpClient();
 
-        public Chart()
+        public Chart(string scheme = null, string host = null, int? port = null)
         {
             Width = 500;
             Height = 300;
             DevicePixelRatio = 1.0;
             Format = "png";
             BackgroundColor = "transparent";
 
-            Scheme = "https";
-            Host = "quickchart.io";
-            Port = 443;
+            if (scheme != null && host != null && port.HasValue)
```

Comment:
Maybe it should be a bit cleverer, e.g.

- if they specify a host only, default to https:443, 
- host and http, set port to 80
- host and https, set port to 443

Reply:
Updated